src/world/lazy.rs

```diff
@@ -29,6 +59,38 @@ pub struct LazyUpdate {
 }
 
 impl LazyUpdate {
+    /// Creates a new `LazyBuilder` which inserts components
+    /// using `LazyUpdate`. This means that the components won't
+    /// be available immediately, but only after a `maintain`
+    /// on `World` is performed.
+    ///
+    /// ## Examples
+    ///
+    /// ```
+    /// # use specs::*;
+    /// # let mut world = World::new();
+    /// struct Pos(f32, f32);
+    ///
+    /// impl Component for Pos {
+    ///     type Storage = VecStorage<Self>;
+    /// }
+    ///
+    /// # let lazy = world.read_resource::<LazyUpdate>();
```

Comment:
I feel like this might be worth exposing in the docs, just for people to understand where LazyUpdate comes from if they haven't seen it elsewhere.

Reply:
I usually hide the fetching since you'll mostly fetch it from a system, whereas in docs examples it's easier to fetch from `World`.